docs/release_notes.rst

```diff
@@ -9,4 +9,17 @@ This page contains the release notes for Qiskit, starting from the point at whic
 "meta-package" structure of Qiskit, see :ref:`legacy-release-notes`.
 
 .. release-notes::
-   :earliest-version: 0.25.0rc1
+   :earliest-version: 1.0.0
```

Comment:
Oh, I guess we should remove this because reno isn't happy there isn't a `1.0.0` tag anywhere yet.
```suggestion
```

Reply:
Oh actually, this will be a problem because if we don't have this set it'll pull in all the historical release notes including all of those in legacy release notes.

Reply:
Thinking about this more, let's try using 0.45.0 here. It won't error but it also won't match a tag on main so it should hopefully only show things > 0.45.0.

```suggestion
   :earliest-version: 0.45.0
```

Reply:
It wasn't happy with this either: https://dev.azure.com/qiskit-ci/qiskit/_build/results?buildId=51437&view=logs&j=abfb3e93-d1ff-5b1b-cb4c-74d6dcf536f9&t=60edc4b3-4259-59c8-ae13-dc31febe2d3d&l=102

I guess let's just go with 0.45.0rc1 for now and we'll just have duplicated entries for 0.45.0. Since this is just dev builds I think that's an ok compromise, and it's better than duplicating all the release history. @ElePT what do you think?